crates/spirv-builder/src/lib.rs

```diff
@@ -265,7 +265,7 @@ fn invoke_rustc(builder: &SpirvBuilder) -> Result<PathBuf, SpirvBuilderError> {
     };
 
     let rustflags = format!(
-        "-Z codegen-backend={} -Zsymbol-mangling-version=v0{}{}",
+        "-Z codegen-backend={} -Zsymbol-mangling-version=legacy{}{}",
```

Comment:
You can just remove it if you want. If you want to keep it in, can you open an issue about switching it back to `v0`, and leave a `// FIXME(#1234) ...` comment here? Thanks.

Reply:
Done and done!